DEVELOP.md

```diff
@@ -0,0 +1,485 @@
+(DFDL) is a language used to describe almost all data formats both
+logically and physically.  DFDL is not a data format or a procedural
+language; rather, it is a data modeling language based on a subset of
+XML Schema annotated with DFDL properties describing the
+representation and layout of each element of the schema inside a
+native text or binary data format.  A DFDL schema allows data to be
+converted between its native data format (physical representation,
+also called a text or binary file) and a DFDL information set (logical
+representation, also called an infoset) such as an XML document, EXI
+document, JSON document, SAX callbacks, or several document object
+model APIs in memory (JDOM, Scala Node, W3C DOM).  When you have a
+DFDL schema for a native data format, you can pick whichever infoset
+type is easiest for you or your application to use and tell a DFDL
+processor to read ("parse") a text or binary file from its native data
+format to that infoset type.  You or your application can do whatever
+you need to do with the infoset and then you can use the same DFDL
+schema and DFDL processor to write ("unparse") the infoset back to its
+native text or binary file format again, completing a round trip from
+native data to infoset to native data again.
+
+Using DFDL avoids inventing a completely new data modeling language,
+avoids writing any parsing and serialization code (with all the bugs
+that normally arise from implementing such code procedurally), and
+makes it much easier to convert any native data format to an infoset,
+operate on the infoset, and convert an infoset back to its native data
+format again.
+
+To learn more about DFDL, you can watch two short
+[videos](https://community.ibm.com/community/user/integration/viewdocument/get-started-with-the-data-format-de)
+put together by Steve Hanson, co-chair of the DFDL working group, read
+a
+[slideshow](https://www.slideshare.net/mbeckerle/dfdl-and-apache-daffodil-overview-from-owl-cyber-defense)
+written by Mike Beckerle, co-chair of the DFDL working group, or go
+through some [tutorials](http://www.xfront.com/DFDL/) written by Roger
+Costello, chair of the DFTVL working group (Data Format Transformation
+and Validation Language, a future not-yet-defined language to specify
+policies for cross-domain system devices).
+
+## Apache Daffodil, IBM DFDL, and ESA DFDL4S
+
+The standards organization in which DFDL started, the Open Grid Forum,
+required 2 implementations in order to move forward with the
+standardization process.  This means that there are two leading DFDL
+processors, a commercial implementation called IBM DFDL bundled into
+IBM's [Integration
+Bus](https://www.ibm.com/docs/en/integration-bus/10.0?topic=model-data-format-description-language-dfdl)
+products and an open source implementation called [Apache
+Daffodil](https://daffodil.apache.org/) hosted by the Apache Software
+Foundation.  The European Space Agency also has created a proprietary
```

Comment:
Apache Software Foundation is introduced here and later the acronym "ASF" is used.  Take the opportunity to associate Apache Software Foundation with its "ASF" acronym  here.